docs/pages/_document.js

```diff
@@ -5,9 +5,24 @@ export default function Document() {
     <Html lang="en">
       <Head>
         <meta charSet="utf-8" />
-        <link rel="icon" href="/icon?<generated>" />
-        <link rel="shortcut icon" href="/favicon.svg" type="image/svg+xml" />
-        <link rel="apple-touch-icon" href="/favicon.svg" />
+        <meta name="viewport" content="width=device-width" />
+
+        {/* PNG favicons as fallback for browsers that don't support SVG */}
+        <link href="/favicon-32x32.png" rel="icon" type="image/png" sizes="32x32" />
```

Comment:
`/docs` prefix is necessary according to [our `basePath` configuration](https://github.com/zeabur/zeabur/blob/e15fe93c2b189df7d95426cd2d3d866d18458ad2/docs/next.config.js#L14)